### ISSUE TITLE
Accessing value on unpublished page causes error

### ISSUE THREAD

**Issue by JJCLane** [Lecoati/LeBlender#31]
If you create a leblender property, save the page (don't publish) and go back to it, you will receive an Object reference is not set to an instance of an object error.

**Comment by soreng** [comment]
Hi @JJCLane could you elaborate a bit on this issue?

Is is because the editor is using data that is taken from the umbraco cache?

**Comment by JJCLane** [comment]
Yep I believe so @soreng, because it can't find it in the cache (because it's not published yet) editors are unable to preview in the backend,

**Comment by soreng** [comment]
So yeah, you can’t/shouldn’t use the cache when rendering in backoffice.

Do you have an example of what you are trying to do?

It may be helpfull for others, and you could get some help fixing your code :-)
